tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/PluginsSDKProjectProvider.java

```diff
@@ -102,6 +103,10 @@ public IStatus doCreateNewProject(
             updateBaseDir = true;
         }
 
+        final boolean hasGradleTools = SDKUtil.hasGradleTools( sdk.getLocation() );
+
+        String arguments = "\"" + projectName + "\"" + " " + "\"" + displayName + "\"";
```

Comment:
The argument escaping shouldn't be handled here.  It should be handled closer to where it is being passed to the OS and only for the create script since the ant script doesn't need the encoding.